source/common/upstream/subset_lb.cc

```diff
@@ -544,7 +544,11 @@ void SubsetLoadBalancer::HostSubsetImpl::update(const HostVector& hosts_added,
   }
 }
 
-HostSetImplPtr SubsetLoadBalancer::PrioritySubsetImpl::createHostSet(uint32_t priority) {
+HostSetImplPtr
+SubsetLoadBalancer::PrioritySubsetImpl::createHostSet(uint32_t priority,
+                                                      uint32_t overprovisioning_factor) {
+  UNREFERENCED_PARAMETER(overprovisioning_factor);
+  // Use original hostset's overprovisioning_factor.
```

Comment:
Should we consider it an error if they're not the same?  ASSERT-validate maybe?

Reply:
You may still need UNREFERENCED_PARAMETER for the opt build to compile, since the ASSERT compiles out there.